File: Patterns in C/Pattern 16.cpp
#include<stdio.h>
int main()
{
    int i,j,n;
    int space=0;
    printf("Enter No of Rows ");
	scanf("%d",&n);
	
    for(i=n; i>=1; i--)
    {
        for(j=1; j<=space; j++)
            printf(" ");

        for(j=1; j<=i; j++)
            printf("%d",j);
        
        for(j=i-1; j>=1; j--)
            printf("%d",j);
            
        printf("\n");
        space++;
    }
    return 0;
}

File: Patterns in Java/Pattern_16.java
import java.util.Scanner;

public class Pattern_16 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int i, j, n, space = 0;
        System.out.print("Enter No of Rows: ");
        n = scan.nextInt();

        for (i = n; i >= 1; i--) {
            for (j = 1; j <= space; j++) {
                System.out.print(" ");
            }
            for (j = 1; j <= i; j++) {
                System.out.print(j);
            }
            for (j = i - 1; j >= 1; j--) {
                System.out.print(j);
            }
            System.out.println();
            space++;
        }
    }
}